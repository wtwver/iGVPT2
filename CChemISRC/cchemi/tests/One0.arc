

                     SUMMARY OF  PM3 CALCULATION, Site No: 1079

                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:09 2020

           Empirical Formula: H2 O  =     3 atoms

 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL
          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV
          CORE-CORE REPULSION     =        149.36282 EV
          GRADIENT NORM           =          0.65369 = 0.37741 PER ATOM
          DIPOLE                  =          1.73858 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316247 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          MOLECULAR WEIGHT        =         18.0152
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.020 SECONDS


          FINAL GEOMETRY OBTAINED
 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
  H     0.00000000 +1   0.77870100 +1  -0.44376600 +1
  H     0.00000000 +1  -0.75754000 +1  -0.46039600 +1
  O     0.00000000 +1   0.00472800 +1   0.10883100 +1
 
