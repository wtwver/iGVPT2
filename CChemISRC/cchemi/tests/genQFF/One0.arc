

                     SUMMARY OF PM3 CALCULATION, Site No: 1079

                                                       MOPAC2012 (Version: 15.146L)
                                                       Sun Dec 13 17:09:22 2015
                                                       No. of days left = 164

           Empirical Formula: H2 O  =     3 atoms

PM3 1SCF CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -37.19905 KCAL/MOL =    -155.64081 KJ/MOL
          TOTAL ENERGY            =       -324.20339 EV
          ELECTRONIC ENERGY       =       -478.05101 EV
          CORE-CORE REPULSION     =        153.84762 EV
          DIPOLE                  =          1.72175 DEBYE    POINT GROUP:       Cs  
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.336209 EV
          HOMO LUMO ENERGIES (EV) =        -12.336  3.890
          MOLECULAR WEIGHT        =         18.015
          COSMO AREA              =         41.96 SQUARE ANGSTROMS
          COSMO VOLUME            =         24.80 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     1.48578
            H     2    O     1     0.53472
            H     2    H     3     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.016 SECONDS
          COMPUTATION TIME        =          0.011 SECONDS


          FINAL GEOMETRY OBTAINED
PM3 1SCF CHARGE=0 Singlet

 Mopac file generated by Gabedit
  O    -0.00966800 +1  -0.37893000 +1   0.00000000 +1
  H     0.81743200 +1   0.21983400 +1  -0.00000300 +1
  H    -0.66411800 +1   0.10784900 +1   0.00000600 +1
 
