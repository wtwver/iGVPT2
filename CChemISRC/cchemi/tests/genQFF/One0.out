 *******************************************************************************
 ** Site#: 1079         For non-commercial use only     Version 15.146L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.146L web: HTTP://OpenMOPAC.net         Days remaining: 164    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            PM3 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Sun Dec 13 17:09:22 2015  *
 *  PM3        - The PM3 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =     0
 *
 *
 *
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM3 1SCF CHARGE=0 Singlet

 Mopac file generated by Gabedit
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O         -0.00966800  *  -0.37893000  *   0.00000000  *
     2       H          0.81743200  *   0.21983400  *  -0.00000300  *
     3       H         -0.66411800  *   0.10784900  *   0.00000600  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         O       -0.0097   -0.3789    0.0000
     2         H        0.8174    0.2198   -0.0000
     3         H       -0.6641    0.1078    0.0000

     ===================== shared-memory parallel CPU Section ======================
     **                                                                           **
     **  MOPAC calculations will run in shared-memory parallel CPUs               **
     **                                                                           **
     **  Number of Threads = 16                                                   **
     **                                                                           **
     **                                                                           **
     **  For information on this work, see:                                       **
     **                                                                           **
     **  Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.        **
     **        DOI: 10.1021/ct3004645.                                            **
     **                                                                           **
     ===================== shared-memory parallel CPU Section ======================
      


  H: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).
  O: (PM3): J. J. P. STEWART, J. COMP. CHEM. 10, 209 (1989).

           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   Cs  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    4


 -------------------------------------------------------------------------------
PM3 1SCF CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM3 CALCULATION
                                                       MOPAC2012 (Version: 15.146L)
                                                       Sun Dec 13 17:09:22 2015
                                                       No. of days left = 164




          FINAL HEAT OF FORMATION =        -37.19905 KCAL/MOL =    -155.64081 KJ/MOL


          TOTAL ENERGY            =       -324.20339 EV
          ELECTRONIC ENERGY       =       -478.05101 EV  POINT GROUP:     Cs  
          CORE-CORE REPULSION     =        153.84762 EV
          COSMO AREA              =         41.96 SQUARE ANGSTROMS
          COSMO VOLUME            =         24.80 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =         12.336209 EV
          HOMO LUMO ENERGIES (EV) =        -12.336  3.890
          NO. OF FILLED LEVELS    =          4
          MOLECULAR WEIGHT        =         18.015

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     3    H     2     1.48578
            H     2    O     1     0.53472
            H     2    H     3     0.00000


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.012 SECONDS
          COMPUTATION TIME        =          0.010 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       O         -0.00966800  *  -0.37893000  *   0.00000000  *
     2       H          0.81743200  *   0.21983400  *  -0.00000300  *
     3       H         -0.66411800  *   0.10784900  *   0.00000600  *

                             CARTESIAN COORDINATES

   1    O       -0.009668000    -0.378930000     0.000000000
   2    H        0.817432000     0.219834000    -0.000003000
   3    H       -0.664118000     0.107849000     0.000006000


           Empirical Formula: H2 O  =     3 atoms



      MOLECULAR POINT GROUP   :   Cs  


                  EIGENVALUES  
 -37.73212 -17.65452 -14.49542 -12.33621   3.88997   5.98369


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          O          -0.358199        6.3582     1.73945     4.61875
    2          H           0.177938        0.8221     0.82206
    3          H           0.180261        0.8197     0.81974
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.140     0.933     0.000     0.944
 HYBRID        -0.090     0.788     0.000     0.793
 SUM            0.051     1.721     0.000     1.722


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  O   1.73945   1.18157   1.43718   2.00000
    2  H   0.82206
    3  H   0.81974

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
